src/runner/tray_icon.cpp

```diff
@@ -3,6 +3,7 @@
 #include "settings_window.h"
 #include "tray_icon.h"
 #include "CentralizedHotkeys.h"
+#include "centralized_kb_hook.h"
```

Comment:
Should we align the naming of these two?

Reply:
I've renamed "CentralizedHotKeys" files to "centralized_hotkeys"